### PR TITLE
Fix unreliability of agent-not-in-property acceptance

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* This causing the tests to fail
~~Sometimes this tests keeps the address data from a previous test because~~
The completeToStep method can go through various routes for the address, this test assumes it will take the user through the manual address step.  This is not always going to happen, sometimes it does an address lookup with `Cr0 2eu`.  Force the test to go through the postcode lookup step

### CODE REVIEW

**Comment by JoeChapman** [comment]
`Sometimes this tests keeps the address data from a previous test` Does it? How does it do that?

**Comment by lennym** [comment]
That worries me too. I'd like to know more about what's going on.

**Comment by sulthan-ahmed** [comment]
@JoeChapman @lennym I think I was wrong, my bad.  I think what happens is that the completeToStep method can go through various routes for the address, e.g. lookup, manual etc

This test assumes it will take the user through the manual address step. This is not always going to happen, sometimes it does an address lookup with Cr0 2eu. As a result, sometimes the test fails when looking for that address.  Force the test to go through the postcode lookup step

**Comment by lennym** [comment]
That makes sense. If left undefined the form filler will pick [one of these](https://github.com/UKHomeOfficeForms/hof-util-testdata/blob/master/lib/data/postcode.json) at random.

The actual problem is that the field that was being specified was `current-property-address` and not `current-property-address-postcode`, which meant that it would enter a postcode from the list above as the postcode, which would probably resolve to an address list, which would be selected from, leaving the address as not the value provided. 

tl;dr - it only worked if the random postcode provided failed an address lookup, which it wouldn't always do.

**Comment by JoeChapman** [comment]
@sulthan-ahmed ok, perhaps you complete the steps to the page manually, i.e. `I.fillField, I.click' so you have more precise control over the input

**Comment by sulthan-ahmed** [comment]
👍
